openedge-plugin/src/main/java/org/sonar/plugins/openedge/sensor/OpenEdgeCodeColorizer.java

```diff
@@ -69,6 +70,8 @@ public void execute(SensorContext context) {
       LOG.debug("Syntax highlight on {}", file.relativePath());
       try {
         highlightFile(context, session, file);
+      } catch (XCodedFileException caught) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Either log or rethrow this exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar.riverside-software.fr/coding_rules#rule_key=squid%3AS1166)